### ISSUE TITLE
Staging Review finding: "Skip to content" link skips banner

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#99116]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Facilities
**Product:** VAMC
**Feature:** Alternative Alert Banners

## Findings details
**VA.gov Experience Standard - issue:** [User does not have a path to the content or feature.](https://design.va.gov/about/experience-standards/#user-has-a-path-to-the-content-or-feature)
**VA.gov Experience Standard - category:** [Findability](https://design.va.gov/about/experience-standards/#category-findability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

The keyboard-accessible "Skip to content" link targets `#content`, which appears after the banner.

Skip links are intended to skip over repetitive content, eg. all the banner and nav stuff that a user may not want to press tab a thousand times to get through. But they shouldn't skip over meaningful or unique content for the page that you're on. When there's no banner present, the skip link makes sense --- skip the header, skip the left nav, get straight to the content. But when there is a banner, there's information that gets passed over entirely.

For a sighted keyboard user the content of the banner is perceivable, but if there's a link in the banner, you have to shift + tab about 15 times to move backwards through the left nav and the breadcrumbs.

For a blind user or a low-vision user with a small viewport, you may never be aware of the content of the banner. The banner does have a heading and it's an accessibly-labeled `region` landmark, which makes it more discoverable for screen reader users. But a user would need to be proactively exploring the page in order to find those.

## Recommended action

As with the heading level issue, this all matches existing behavior and is consistent with banners on the homepage. I don't have a specific recommendation here, except that this is an area for future conversation and hopefully finding a new approach.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Keyboard
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jilladams** [comment]
@laflannery @Agile6MSkinner same here: should we close this as a no-op that we will not pursue, or should we open a Sitewide ticket to track it longer term?

**Comment by laflannery** [comment]
Same as the other ticket - how should the Skip to content link function overall on VA.gov for all pages when there is a banner vs when there isn't a banner? And does this change when there is a banner on a page with a secondary nav? (It might because then the user might have to tab through all those links). It's a much larger discussion it's just that we happened to be the team that brought this to the attention of Governance
